app/javascript/app/pages/login/login-component.jsx

```diff
@@ -17,9 +17,13 @@ class Login extends Component {
       email: null
     };
   }
-
   render() {
     const { dispatch, handleLoginThunk } = this.props;
+    const { email, password } = this.state;
+    const handleLogin = () => dispatch(handleLoginThunk(password, email));
+    const handleKeyUp = e => {
```

Comment:
Same than before, defining functions on the render lifecycle doesn't sound good.